extensions/data-workspace/src/common/strings.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
should this file be named constants.ts or localizedConstants.ts like other extensions have?